<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TalkWindow</class>
 <widget class="QWidget" name="TalkWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1006</width>
     <height>616</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
background-color: rgb(255, 255, 255);
</string>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>2</y>
      <width>1000</width>
      <height>610</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1001</width>
       <height>611</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,11">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1">
          <item>
           <widget class="QFrame" name="frame">
            <property name="styleSheet">
             <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(156, 100, 255, 255), stop:1 rgba(151, 220, 255, 255));</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLabel" name="name">
             <property name="geometry">
              <rect>
               <x>420</x>
               <y>0</y>
               <width>161</width>
               <height>41</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">background:transparent;
font: 16pt &quot;黑体&quot;;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
            <widget class="QPushButton" name="hide">
             <property name="geometry">
              <rect>
               <x>920</x>
               <y>0</y>
               <width>38</width>
               <height>38</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{background-color:transparent;font: 18pt &quot;楷体_GB2312&quot;;border-radius: 3px; color: rgb(255, 255, 255);}
QPushButton:hover { background-color: rgb(255, 100, 100); }</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="image.qrc">
               <normaloff>:/image/min.png</normaloff>:/image/min.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QPushButton" name="close">
             <property name="geometry">
              <rect>
               <x>959</x>
               <y>0</y>
               <width>38</width>
               <height>38</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{background-color:transparent;font: 18pt &quot;楷体_GB2312&quot;;border-radius: 3px; color: rgb(255, 255, 255);}
QPushButton:hover { background-color: rgb(255, 100, 100); }</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="image.qrc">
               <normaloff>:/image/close.png</normaloff>:/image/close.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="70,1,10,30,10">
          <item>
           <widget class="QTextBrowser" name="msgBrowser">
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 100%);
background:transparent;border-width:0;border-style:outset

</string>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAsNeeded</enum>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(156, 100, 255, 255), stop:1 rgba(151, 220, 255, 255));</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="lineWidth">
             <number>15</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_4" native="true">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);</string>
            </property>
            <widget class="QToolButton" name="toolButton_3">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>0</y>
               <width>30</width>
               <height>30</height>
              </rect>
             </property>
             <property name="toolTip">
              <string>表情</string>
             </property>
             <property name="styleSheet">
              <string notr="true">background:transparent;</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="image.qrc">
               <normaloff>:/image/expression.png</normaloff>:/image/expression.png</iconset>
             </property>
            </widget>
            <widget class="QFontComboBox" name="fontCbx">
             <property name="geometry">
              <rect>
               <x>60</x>
               <y>0</y>
               <width>151</width>
               <height>31</height>
              </rect>
             </property>
             <property name="toolTip">
              <string>字体</string>
             </property>
            </widget>
            <widget class="QComboBox" name="sizeCbx">
             <property name="geometry">
              <rect>
               <x>216</x>
               <y>1</y>
               <width>91</width>
               <height>31</height>
              </rect>
             </property>
             <property name="toolTip">
              <string>字体大小</string>
             </property>
             <item>
              <property name="text">
               <string>5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>9</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>10</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>11</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>12</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>13</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>14</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>16</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>18</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>20</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>22</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>24</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>26</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>28</string>
              </property>
             </item>
            </widget>
            <widget class="QToolButton" name="boldTbtn">
             <property name="geometry">
              <rect>
               <x>310</x>
               <y>0</y>
               <width>30</width>
               <height>30</height>
              </rect>
             </property>
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;加粗&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="styleSheet">
              <string notr="true">background:transparent;</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="image.qrc">
               <normaloff>:/image/bold.png</normaloff>:/image/bold.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="arrowType">
              <enum>Qt::NoArrow</enum>
             </property>
            </widget>
            <widget class="QToolButton" name="italicTbtn">
             <property name="geometry">
              <rect>
               <x>340</x>
               <y>0</y>
               <width>30</width>
               <height>30</height>
              </rect>
             </property>
             <property name="toolTip">
              <string>斜体</string>
             </property>
             <property name="styleSheet">
              <string notr="true">background:transparent;</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="image.qrc">
               <normaloff>:/image/italic.png</normaloff>:/image/italic.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QToolButton" name="underlineTbtn">
             <property name="geometry">
              <rect>
               <x>370</x>
               <y>0</y>
               <width>30</width>
               <height>30</height>
              </rect>
             </property>
             <property name="toolTip">
              <string>下划线</string>
             </property>
             <property name="styleSheet">
              <string notr="true">background:transparent;</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="image.qrc">
               <normaloff>:/image/underline.png</normaloff>:/image/underline.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QToolButton" name="colorTbtn">
             <property name="geometry">
              <rect>
               <x>400</x>
               <y>0</y>
               <width>30</width>
               <height>30</height>
              </rect>
             </property>
             <property name="toolTip">
              <string>字体颜色</string>
             </property>
             <property name="styleSheet">
              <string notr="true">background:transparent;</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="image.qrc">
               <normaloff>:/image/color.png</normaloff>:/image/color.png</iconset>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
            <widget class="QToolButton" name="saveTbtn">
             <property name="geometry">
              <rect>
               <x>430</x>
               <y>0</y>
               <width>30</width>
               <height>30</height>
              </rect>
             </property>
             <property name="toolTip">
              <string>保存</string>
             </property>
             <property name="styleSheet">
              <string notr="true">background:transparent;</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="image.qrc">
               <normaloff>:/image/save.png</normaloff>:/image/save.png</iconset>
             </property>
            </widget>
            <widget class="QToolButton" name="clearTbtn">
             <property name="geometry">
              <rect>
               <x>460</x>
               <y>0</y>
               <width>30</width>
               <height>30</height>
              </rect>
             </property>
             <property name="toolTip">
              <string>清空</string>
             </property>
             <property name="styleSheet">
              <string notr="true">background:transparent;</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="image.qrc">
               <normaloff>:/image/clear.png</normaloff>:/image/clear.png</iconset>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="msgTextEdit">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
background:transparent;border-width:0;border-style:outset</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <widget class="QWidget" name="horizontalLayoutWidget_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>995</width>
               <height>32</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="10,1,0,0,0">
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="close2">
                <property name="minimumSize">
                 <size>
                  <width>80</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
font: 10pt &quot;黑体&quot;;background-color:rgb(235, 235, 235);border-radius: 3px; color: rgb(0, 0, 0);}
QPushButton:hover { background-color: rgb(225, 225, 225); }</string>
                </property>
                <property name="text">
                 <string>关闭</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>10</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="minimumSize">
                 <size>
                  <width>110</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{font: 10pt &quot;黑体&quot;;background-color:rgb(0, 170, 255);border-radius: 3px; color: rgb(0, 0, 0);}
QPushButton:hover { background-color: rgb(0, 150, 255); }</string>
                </property>
                <property name="text">
                 <string>发送</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>15</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
             <zorder>pushButton</zorder>
             <zorder>close2</zorder>
             <zorder>horizontalSpacer</zorder>
             <zorder>horizontalSpacer_3</zorder>
             <zorder>horizontalSpacer_2</zorder>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
